.github/workflows/deploy.yml

```
@@ -0,0 +1,45 @@
+  pages: write
+  id-token: write
+
+concurrency:
+  group: pages
+  cancel-in-progress: false
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

actionlint (1.7.7)

property "deployment" is not defined in object type {}

(expression)